app/views/places/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1> <%= @places.name %> - places details</h1>
+
+<table class="table table-striped">
+  <thead>
+    <tr>
+      <th scope="col"><%= @places.name %></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>City of training</td>
+      <td><%= @city.nameofcity %></td>
```

Comment:
I think we're missing this `@city` variable in the controller

Reply:
done